ripple/src/main/webapp/landing.js

```diff
@@ -33,9 +33,9 @@ function searchAddressAutocomplete() {
     });
 }
 
+var ENTER_KEY = 13;
```

Comment:
There's 2 ENTER_KEY global variables now. Could we consolidate to just having one?

Reply:
Yes, sorry I didn't realize it was in a different file!